docs/data/osdf/install-cache-rpm.md

```diff
@@ -19,8 +19,7 @@ Before starting the installation process, consider the following requirements:
 * __Host certificate:__ Required for authentication.  See note below.
 * __Network ports:__ The cache service requires the following ports open:
   * Inbound TCP port 8443 for file access via the HTTP(S) and XRoot protocols.
-  * (Optional) Inbound TCP port 8444 for access to the web interface for monitoring and configuration;
-    if enabled, access to this port should be restricted to the LAN.
+  * Inbound TCP port 8444 for access to the web interface for monitoring and configuration.
```

Comment:
This is a security hazard until https://github.com/PelicanPlatform/pelican/issues/1227 is solved.

Reply:
@matyasselmeci please make the suggestion to revert, apply it, then go ahead and merge this PR. We can sort out this detail at a future date.